cpp/cmake/versions.json

```
@@ -1,9 +1,11 @@
 {
   "packages" : {
-      "Thrust" : {
-        "version" : "1.15.0",
-        "git_url" : "https://github.com/NVIDIA/thrust.git",
-        "git_tag" : "${version}"
+    "Thrust" : {
+      "version" : "1.12.0",
```

```suggestion
      "version" : "1.15.0",
```

All of RAPIDS has been using 1.15 for releases 22.02 and 22.04, though looking at the CMake files this versions file is not being used to override versions, so it might be good to get rid of it entirely for now?